### PR TITLE
Add the missing steps in the autotune_installation

### PR DESCRIPTION
**Pull request by rpancham**
This PR fixes the issue https://github.com/kruize/autotune/issues/807

Added the missing installation steps of kubectl and minikube

### CODE REVIEW

**Comment by msvinaykumar** [comment]
LGTM

**Review by msvinaykumar — APPROVED** [review]

File: docs/autotune_install.md [hunk 1]
```diff
@@ -31,8 +31,30 @@ Autotune can be deployed to a supported Kubernetes cluster. We currently support
 
 ### Install kubectl and minikube
 
+**Install kubectl**
+
+**`curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl"
```
Comment (chandrams) [inline docs/autotune_install.md:36]: User should be able to copy these commands and paste it. Please update the entire read me wherever commands are present

Comment (chandrams) [reply docs/autotune_install.md:36]: As discussed, also update the deploy.sh with the latest usage options and output.

File: docs/autotune_install.md [hunk 2]
```diff
@@ -66,6 +88,20 @@ $ ./build.sh -i autotune:test
...
+
+click here to create an account in docker hub [Docker](https://hub.docker.com/)
+
+**`docker login`**
+
+**`docker tag autotune:test docker.io/username/autotune:test`**
+
+2.Push the Docker image
+
+**`docker push uname/autotune:test`**
```
Comment (chandrams) [inline docs/autotune_install.md:103]: Please change uname to <username> to keep it consistent

**Pull request merged**